package.json

```diff
@@ -26,7 +26,8 @@
     "loopback-connector": "^2.2.0",
     "mongodb": "~2.0.35",
     "async": "^1.0.0",
-    "debug": "^2.1.1"
+    "debug": "^2.1.1",
+    "kerberos": "0.0.21"
```

Comment:
If the peer dependency says `kerberos@~0.0`, then why are we fixing `0.0.21` as our version? Can we use `"kerberos": "~0.0.21"` instead, so that we automatically update when a bugfix is released?